manager/controllers/app/fybrikmodule_controller.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package app
+
+import (
+	"context"
+	"fmt"
+
+	"encoding/json"
+
+	validate "fybrik.io/fybrik/pkg/taxonomy/validate"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	"github.com/go-logr/logr"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+
+	api "fybrik.io/fybrik/manager/apis/app/v1alpha1"
+	"fybrik.io/fybrik/manager/controllers/utils"
+)
+
+// FybrikModuleReconciler reconciles a FybrikModule object
+type FybrikModuleReconciler struct {
+	client.Client
+	Name   string
+	Log    logr.Logger
+	Scheme *runtime.Scheme
+}
+
+// Reconcile reconciles FybrikModule CRD
+// It receives FybrikModule CRD and selects the appropriate modules that will run
```

Comment:
These comments look like they were cut and pasted.  They don't appear to be a description of what this controller does.

Reply:
Modified to be specific to Fybrik Module